src/gleam/map.gleam

```diff
@@ -5,6 +5,12 @@ if javascript {
   import gleam/pair
 }
 
+if javascript {
+  // hack to include another js file..
+  pub external fn include_persistent_hash_map() -> Nil =
+    "../persistent-hash-map.mjs" "__include_me"
```

Comment:
Is this planned to be moved into gleam_stdlib.mjs?

Reply:
Maybe. I would prefer to keep it separate from a maintenance perspective but probably needs a better way to include it. @lpil thoughts?

Reply:
Sorry for the slow reply, I have been on holiday.

Why do we need to include it here? It looks like this file is imported into `gleam_stdlib.mjs` already?

Reply:
iirc when importing only from `gleam_stdlib.mjs` the file wasn't copied to the build output folder. Import from a .gleam file and it gets copied.

Reply:
We copy everything, don't need to import anything into Gleam for it to to work